### ISSUE TITLE
ChunkIterator not iterating correctly

### ISSUE THREAD

**Issue by rhasselbaum** [lemire/javaewah#59]
Similar to issue #58, I am seeing incorrect behavior in `ChunkIterator` in release 1.1.2. Sample code:

```
import com.googlecode.javaewah.*;

EWAHCompressedBitmap bitmap = EWAHCompressedBitmap.bitmapOf(243,260,1000);
ChunkIterator iter = bitmap.chunkIterator();
iter.move(245);
assert iter.nextLength() == 15;  // fails; returns 691
```

**Comment by gssiyankai** [comment]
Thanks @rhasselbaum for reporting the issue.
Fixed @497c2e2b44c0963474779ff57085fd16181f5e55

**Comment by lemire** [comment]
@gssiyankai 

According to travis, we have test failures: 

https://travis-ci.org/lemire/javaewah

Do our tests pass for you?

**Comment by gssiyankai** [comment]
@lemire,
My bad. It should be okay now :)

**Comment by rhasselbaum** [comment]
Thank you! After pull from master, my tests are passing now, too.

**Comment by lemire** [comment]
Fix in release 1.1.3. Closing.
